reports/inventory/helpers.ts

```diff
@@ -81,6 +89,7 @@ export function getStockLedgerEntries(
       item,
       location,
 
+      batchNumber,
```

Comment:
Minor nitpick but this should be grouped with `item` and `location`; those new lines between properties aren't random.

Reply:
corrected